### PR TITLE
Kcho/requirements

### PR DESCRIPTION
**Pull request by kcho**
`requirements.txt` file was dropped in one of our previous PRs, but we need this file to help Pronet install the `dev` version of Lochness.

### CODE REVIEW

**Comment by kcho** [comment]
Please review this PR before PR #73

**Comment by tashrifbillah** [comment]
Sure let's discuss this once more. `requirements.txt`  is not used anywhere. Whether you do `git clone` or `pip install .`, always `setup.py` is used.

So the question I am asking from the beginning about `requirements.txt` is that, where is it used?

**Comment by kcho** [comment]
I'm using `pip install -r requirements.txt` which I thought is more conventional? If `pip install .` works exactly the same as the `pip install -r requirements.txt` I shall remove this PR and update documentation. (I don't want them to create another build directory apart from the raw repository pulled from github) Let me know please.

**Comment by tashrifbillah** [comment]
> I don't want them to create another build directory

Testing this ...

**Comment by tashrifbillah** [comment]
Okay I agree, with `pip install -r requirements.txt`, the three directories `dist`, `build`, and `lochness.egg_info` are not created. But installing the requirements.txt is not enough for this project because most updates are in `lochness` itself. To avail the latter, we must do:

```
git clone https://https://github.com/AMP-SCZ/lochness.git
cd lochness
pip install .
```
or
```
pip install git+https://https://github.com/AMP-SCZ/lochness.git
```
Since we don't want to three additional directories, we should use the latter method. Above all, the latter method does not use `requirements.txt`. It uses `setup.py`.

So we should just forget this PR. Let me know if you have found otherwise.

**Comment by kcho** [comment]
> But installing the requirements is not enough for this project because most updates are in lochness itself. To avail the latter, we must do:

Yes, for this we are adding the cloned `lochness` directory to `$PYTHONPATH`, and `lochness/scripts` to `$PATH` for clean dev purposes, which does not require `lochness` to be built in different locations. It's easier to go along with the most recent updates in scripts (with simple `git pull`) for dev purposes.

**Comment by tashrifbillah** [comment]
Okay I understand. I am working on proposals.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+paramiko>=2.7.2
+pandas>=1.3.2
+boxsdk>=2.11.0
+requests>=2.26.0
+pytest>=6.2.4
+git+https://github.com/AMP-SCZ/yaxil@main#egg=yaxil
```
Comment (tashrifbillah) [inline requirements.txt:6]: Why is it not `'yaxil>=0.5.2'` like that of setup.py?

Comment (kcho) [reply requirements.txt:6]: I haven't got to the `setup.py` the same way as `requirements.txt` (to use `AMP-SCZ` yaxil yet) Do you know how to structure `setup.py` to install from specific repository? We need to install `yaxil` under AMP-SCZ. (I had submitted a PR to `harvard-nrg/yaxil` but it has not been reviewed yet.)

Comment (tashrifbillah) [reply requirements.txt:6]: Yes I am working on the way.

**Review by tashrifbillah — APPROVED** [review]

**Pull request merged**